### PR TITLE
Fix BigInteger explicit cast

### PR DESCRIPTION
**Pull request by erikzhang**

### CODE REVIEW

File: src/Neo.Compiler.CSharp/MethodConvert.cs [hunk 1]
```diff
@@ -3195,8 +3195,8 @@ private bool TryProcessSystemMethods(SemanticModel model, IMethodSymbol symbol,
                             PrepareArgumentsForMethod(model, symbol, arguments);
                         JumpTarget endTarget = new();
                         AddInstruction(OpCode.DUP);
-                        Push(-128);
-                        Push(128);
+                        Push(sbyte.MinValue);
+                        Push(sbyte.MaxValue);
```
Comment (erikzhang) [inline src/Neo.Compiler.CSharp/MethodConvert.cs:3199]: The values are different. We must revert this commit.

**Review by shargon — DISMISSED** [review]

File: src/Neo.Compiler.CSharp/MethodConvert.cs (hunk 1 above)
Comment (shargon) [reply src/Neo.Compiler.CSharp/MethodConvert.cs:3199]: But it should be the same, isn't it?

Comment (erikzhang) [reply src/Neo.Compiler.CSharp/MethodConvert.cs:3199]: https://github.com/neo-project/neo-vm/blob/59b8ac73d285aa9d607aee53eeadde82fcba9324/src/neo-vm/ExecutionEngine.cs#L958-L965

Comment (erikzhang) [reply src/Neo.Compiler.CSharp/MethodConvert.cs:3199]: The `WITHIN` opcode checks the range `[a, b)`.

Comment (shargon) [reply src/Neo.Compiler.CSharp/MethodConvert.cs]: Please check it again

Comment (erikzhang) [reply src/Neo.Compiler.CSharp/MethodConvert.cs]: Good.

**Review by shargon — APPROVED** [review]

**Pull request merged**